src/cascadia/TerminalSettingsEditor/NullableColorPicker.xaml

```diff
@@ -0,0 +1,154 @@
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(6), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(7), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(8), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(9), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(10), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(11), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(12), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(13), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(14), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                    <ContentControl Content="{x:Bind ColorEntryAt(15), Mode=OneWay}"
+                                    ContentTemplate="{StaticResource ColorPreviewChipTemplate}"
+                                    IsTabStop="False" />
+                </VariableSizedWrapGrid>
+            </DataTemplate>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <Grid ColumnSpacing="5">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+
+        <ContentDialog x:Name="ColorPickerDialog"

```

Comment:
content dialogs can be spicy - test this on Win10!

Reply:
specifically content dialogs inside grids even

Reply:
On Win10:
![image](https://github.com/user-attachments/assets/4f8c2c82-c507-4270-8894-ab3ce5d0bf5d)

🤢🤢🤢

The text boxes don't work either.

And actually, I was curious and played around with it a bit in Win11. That gives us a _new_ set of issues:
- text boxes don't work in the ColorPicker control when it's in a ContentDialog
- sometimes the focus jumps _behind_ the ContentDialog

UGH, this might require a redesign on my end _again_. I'll revisit this in a bit.

Reply:
Yikes

Reply:
Fixed and validated. See https://github.com/microsoft/terminal/pull/17870#issuecomment-2460637117